### PR TITLE
Use SQLAlchemy attribute events instead of attribute extension

### PR DESCRIPTION
**Pull request by youtux**

### CODE REVIEW

File: traduki/sqla.py [hunk 1]
```diff
@@ -120,37 +122,33 @@ def _do_compare(self, op, other, escape):
...
 
-            :note: The value is copied using dict to avoid 2 objects
-                referring to the same Translation. Also the oldvalue should
-                wipe the values for the languages that are not in the value.
+        :param state: SQLAlchemy instance state.
+        :param value: The value that is being assigned.
+        :param oldvalue: The current value.
+        :param initiator: SQLAlchemy initiator (accessor).
+        """
+        if value is None or value in (NO_VALUE, NEVER_SET):
```
Comment (blaise-io) [inline traduki/sqla.py]: For future compatibility, maybe check `value` against supported types instead?

File: traduki/sqla.py [hunk 2]
```diff
@@ -172,12 +169,31 @@ def i18n_relation(column=None, comparator_factory=TranslationComparator,
...
+        res = relationship(
+            Translation,
+            comparator_factory=comparator_factory,
+            lazy=lazy,
+            # This required only for sqlalchemy < 1.3.4. See https://github.com/sqlalchemy/sqlalchemy/issues/4691
+            active_history=True,
+            **kwargs
+        )
+
+        # We must attach the event on before_configured, otherwise it is too late if we use after_configured, as other
```
Comment (blaise-io) [inline traduki/sqla.py]: This copy is a bit awkward, maybe change it to something like _We must attach the event on before_configured. If we use after_configured, we're too late as other hooks will already be installed by SQLAlchemy._

File: traduki/sqla.py (hunk 1 above)
Comment (youtux) [reply traduki/sqla.py]: done

File: traduki/sqla.py (hunk 2 above)
Comment (youtux) [reply traduki/sqla.py]: done

**Pull request merged**